modules/06 Angular/Assignments/12/answer12/src/app/actor/actor.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component} from '@angular/core';
+import { Actor } from './actor.model';
+
+@Component({
+  selector: 'app-actor',
+  templateUrl: './actor.component.html',
+  styleUrls: ['./actor.component.css']
+})
+
+
+export class ActorComponent {
+  actors: Actor[] = null;
+
+  constructor() {
+    this.actors = [
```

Comment:
You could do this right in place you've defined the variable, you don't need the constructor for this